### DISCUSSION TITLE
An error occurred when running webxr, please help

### DISCUSSION THREAD

**Discussion by liuhuzhong** [cefsharp/CefSharp#4055] [General]
I used cefsharp100.0.4896.60 to make a browser of my own, but I can’t use webxr functions, such as opening the following webpage: https://immersive-web.github.io/webxr-samples/immersive-vr-session.html when I click the Enter XR button Does the prompt "XRSession Creation failed" require additional settings? Please give a reminder from friends who have used it, thank you very much! ! (I am using a vive helmet-mounted VR device, which is normally used under Google Chrome!)

**Comment by amaitland** [comment]
Duplicate of #3330
